Dockerfile

```diff
@@ -1,119 +1,34 @@
-FROM manageiq/ruby
-MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
-
-## For SUI
-ARG REF=master
+FROM centos/httpd:latest
 
-## Set ENV, LANG only needed if building with docker-1.8
-ENV container=docker \
-    CONTAINER=true \
-    APACHE_CONF_DIR=/etc/httpd/conf.d \
-    APP_ROOT=/var/www/miq/vmdb \
-    PERSISTENT=/persistent \
-    APPLIANCE_ROOT=/opt/manageiq/manageiq-appliance \
-    SUI_ROOT=/opt/manageiq/manageiq-ui-service \
-    CONTAINER_SCRIPTS_ROOT=/opt/manageiq/container-scripts \
-    TERM=xterm
+MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
 
 ## Atomic/OpenShift Labels
 LABEL name="manageiq-apache" \
       vendor="ManageIQ" \
       version="2.4.6-45" \
```

Comment:
Let's drop this version label, we'll be getting out of sync with the centos/httpd  defined one as per https://github.com/CentOS/CentOS-Dockerfiles/blob/master/httpd/centos7/Dockerfile